rpc/common/moc_common_common.proto

```diff
@@ -343,3 +343,15 @@ enum CloudInitDataSource {
 	NoCloud = 0;
 	Azure = 1;
 }
+
+enum NetworkType {
```

Comment:
Shouldn't we already have this enum?

Reply:
we have the virtual network type before, I move it from virtual network to common, so that logical network can use it